README.md

```diff
@@ -2,12 +2,66 @@
 
 A registry for questionnaire schemas for eq-questionnaire-runner.
 
+
+## Setup
+
+It is recommended that you use [Pyenv](https://github.com/pyenv/pyenv) to manage your Python installations.
+
+### Install Poetry
+
+We use poetry to manage the dependencies in this repository, to install poetry use:
+
+```
+curl -sSL https://install.python-poetry.org | python3 -
+```
+
+Install the dependencies for this project using
+```
+poetry install
+```
+
+## Validating Schemas
+
 A make target exists (`validate-schemas`) in order to validate schemas locally using eq-questionnaire-validator docker image.
 
 To do this run `make run-validator` to run the validator and then run `make validate-schemas` to validate the schemas.
 
 **Schema file names must use snake case to be compatible with runner**
 
+## Translations
+
+** N.B. Currently only supporting translations for Health themed surveys (PHM).
+
+Ensure that the latest release of the eq-translations repository has been installed before running any of the following scripts (the release version is checked automatically when running translations, scripts will not run if the release is out of date.)
+
+### Generating Translation Templates
+
+To generate a translation template `.pot` file in order to translate a schema, use the following command. It will generate a template file containing all the strings to be translated:
+```
+make translation-templates
+```
+
+### Generating Translated Schemas
+
+To translate a schema, a `.po` file for the schema will need to be added to the `translations/{THEME}/{LANGUAGE_CODE}` directory containing strings from the schema
+and the matching translations. 
+
+The `.po` file needs to be named in the following format: `{SCHEMA_NAME}_{LANGUAGE_CODE}.po`
```

Comment:
If it is in a language specific folder, then why does the filename need the language code?. I would drop the code that requires it, as you should already be in the context of a language folder.

Reply:
Removed this 👍